static/bidder-info/gothamads.yaml

```diff
@@ -0,0 +1,14 @@
+endpoint: "http://us-e-node1.gothamads.com/?pass={{.AccountID}}"
+maintainer:
+  email: "support@gothamads.com"
```

Comment:
@gotham-ads , email is being sent from prebid team to verify email address. Requesting  to respond back to complete verification.

Reply:
replied

Reply:
received response